packages/jetpack-protect/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "automattic/jetpack-protect",
+	"description": "Protect library for Jetpack",
+	"homepage": "http://jetpack.com/",
+	"version": "7.2.2",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"support": {
+		"issues": "https://github.com/Automattic/jetpack/issues"
+	},
+	"autoload": {
+		"psr-4": {
+			"Jetpack\\V7\\Protect\\": "lib/"
+		},
+		"files": [
+			"module.php"
+		]
+	},
+	"require": {
+		"php": ">=5.3.3",
+		"automattic/jetpack-core": "7.2.2"
+	},
+	"require-dev": {
+		"dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
+		"wp-coding-standards/wpcs": "2.0.0",
+		"sirbrillig/phpcs-variable-analysis": "2.6.0",
+		"phpcompatibility/phpcompatibility-wp": "2.0.0"
+	},
+	"scripts": {
+		"php:compatibility": "composer install && vendor/bin/phpcs -p --runtime-set testVersion '5.3-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
+		"php:lint": "composer install && vendor/bin/phpcs -p",
+		"php:autofix": "composer install && vendor/bin/phpcbf",
+		"php:lint:errors": "composer install && vendor/bin/phpcs -p --runtime-set ignore_warnings_on_exit 1"
```

Comment:
Shouldn't linting (and other development-oriented tasks) happen from root `composer.json` in a monorepo approach? Maintaining those commands individually for each package might otherwise get pretty funky. :-)

Reply:
Oh, this is a very good question that I happen to be struggling with currently. How would you say we should be testing, for example? What part of the repository should list phpunit as its dependency, and how do we run phpunit against a particular phpunit.xml?
I'd appreciate any insight you have on the matter!